src/TileSlider.tsx

```diff
@@ -155,16 +152,22 @@ export const TileSlider = <T,>({
     }
 
     return index;
-  }, [cycleMode, getSliderWidth, isMultiPage, items.length, tilesToShow]);
+  }, [cycleMode, isMultiPage, items.length, tilesToShow]);
 
   const getSliderPosition = useEventCallback(() => {
-    const transform = getComputedStyle(frameRef.current).transform?.split(', ')[4];
+    const transform = frameRef.current ? getComputedStyle(frameRef.current).transform?.split(', ')[4] : '0';
 
     return transform ? parseInt(transform) : 0;
   });
 
+  const handleResize = useEventCallback(() => {
+    cancelAnimationFrame(sliderDataRef.current.animationId);
+    sliderDataRef.current.frameWidth = frameRef.current ? parseFloat(getComputedStyle(frameRef.current).width) : 0;
+    frameRef.current.style.transform = `translateX(${-responsiveTileWidth * state.index}%)`;
```

Comment:
If `frameRef.current` can be nullish, then `.style.transform` would also fail?

Reply:
Good point!